com.playeveryware.eos/CHANGELOG.md

```diff
@@ -4,6 +4,282 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+  - fix: Add to gitignore to avoid init scene being added.
+  - fix: Correct the function used to convert byte array to string by providing the start index and the number of bytes to read.
+  - fix: Change function signature in EnumUtility to 'GetEnumerator'.
+  - fix: Re-disable the new runtime config via scripting defines.
+  - fix: Connect IntegratedPlatformManagementFlags field member to the RuntimeConfig data structure.
+  - fix(network): Explicit double value as 0.0, Settings debug logs warning when string is invalid
+  - fix (sessions): Non-Owners of Sessions Cannot Manage State
+  - fix: Change SandboxId to be string instead of Guid.
+  - fix: Move conversion from EOSConfig to RuntimeConfig from within EOSConfig to being an implicit conversion operator within RuntimeConfig.
+  - fix: Add comments to enum utility function.
+  - fix: Correct order of parsing operations.
+  - fix: Correct strings for descriptions of the flag values.
+  - fix: Remove unreferenced code path and member within PlatformConfig.
+  - fix: Add comment to IsEncryptionKeyValid() method.
+  - fix: Correct the format of comments for EOSConfig field members.
+  - fix: Correct signature used for configuring the override thread affinity values.
+  - fix: Re-introduce public methods to EOSManager that were removed.
+  - fix: Remove unused function that Loads EOSConfig into memory.
+  - fix: Correct comment to properly describe GetULongFromString method.
+  - fix: Correct scripting defines to support the EOS_DISABLE flag.
+  - fix: Move functionality to parse and validate Enum string flags to dedicated place for such tasks.
+  - fix: Move SafeTranslatorUtility to Core (from Editor) to fit with new asmdef files surrounding test assemblies.
+  - fix (sessions): Non-Owners of Sessions Cannot Manage State
+  - fix: Rename window and config, move both to appropriate place within the editor assembly.
+  - fix: Rename unit test configuration window for clarity, and disabled access to the unit test results parser.
+  - fix: add discard to render async call.
+  - fix: Add Serializable attribute to PackagingConfig class.
+  - fix: Remove call to set the window title for PluginVersionWindow, as it is explicitly set during the render content step.
+  - fix: Move DLL signing utility stuff into SigningUtility, remove SigningConfigEditor, after adding RenderInput support for List<string>
+  - fix(connect,apple) : correct modified class names, and remove unwanted MonoBehaviour
+  - fix (EOSManager): Don't use authToken if it is null
+
+- Git Workflow/Action
+  - fix: Change format of README.md so that multiple GitHub badges are rendered side-by side and left-aligned.
+  - fix: Change color of Unity version badge to be blue.
+  - fix: Add conditional so that if the unity version badge is already accurate the workflow does not fail.
+  - fix: Update the version of the checkout action to avoid warnings regarding running on a deprecated version of nodejs.
```

Comment:
I don't think any of this git workflow/action things are needed for the plugin changelog.